src/js/components/InspectorSidebar.tsx

```diff
@@ -93,8 +90,7 @@ class BaseInspector extends React.Component<Inspector> {
         ctor = 'Scale';
         primType = TYPES.SCALES;
       }
-
-      // InspectorType = Inspector[ctor]; // TODO: This throws an error when uncommented
+      // InspectorType = Inspector[ctor]; // TODO: This throws an error when uncommented, unkown reason
```

Comment:
was this already here when you got to it, alan? can we add a note to this TODO assigning it to arvind if so